src/components/SyncContinouslyPendingOperations.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+// Sync continuously the accounts that have pending operations
+
+import React, { Component } from 'react'
+import logger from 'logger'
+import { createStructuredSelector, createSelector } from 'reselect'
+import { connect } from 'react-redux'
+import type { Account } from '@ledgerhq/live-common/lib/types'
+import { BridgeSyncConsumer } from 'bridge/BridgeSyncContext'
+import type { Sync } from 'bridge/BridgeSyncContext'
+import { accountsSelector } from 'reducers/accounts'
+
+const accountsWithPendingOperationsSelector = createSelector(accountsSelector, accounts =>
+  accounts.filter(a => a.pendingOperations.length > 0),
+)
+
+class SyncContPendingOpsConnected extends Component<{
+  sync: Sync,
+  accounts: Account[],
+  priority: number,
+  interval: number,
+}> {
+  componentDidMount() {
+    this.timeout = setTimeout(this.check, this.props.interval)
+  }
+  componentWillUnmount() {
+    clearTimeout(this.timeout)
+  }
+  check = () => {
+    const { sync, accounts, priority, interval } = this.props
+    setTimeout(this.check, interval)
```

Comment:
means that we always push new checks even if sync is not finished?
orr is it handled at lower level?